### DISCUSSION TITLE
The solution in push code when git user info changed.

### DISCUSSION THREAD

**Discussion by popstar7** [Finalgoal231/finalGoal#87] [About github] [answered]
When I changed user info with "git config user.email example@mail.com", the info will not change in fact.
Let's discuss to solve this.

**Comment by uneoka0114** [comment]
By setting up `post-receive` hook, you can automate the deployment of your proxy server code to GitHub whenever someone pushes new code to your repository.

**Comment by noahstoner** [comment, marked as answer]
Enter to Control Panel.
User accounts -> Credential Manager -> Windows Credentials
The old git user info saved there.
First remove this old info, then push again.
Then the authentication popup will be display.
